src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -114,7 +117,8 @@ export default class VivViewerLayer extends CompositeLayer {
             (!viewportId || this.context.viewport.id === viewportId)),
         z: numLevels - 1,
         pickable: true,
-        onHover
+        onHover,
+        boxSize: 2 ** Math.ceil(Math.log2(Math.max(lowResWidth, lowResHeight)))
```

Comment:
Can we move this to a function with a better name rather than in-lining each time?